deploy/runtools/firesim_topology_elements.py

```diff
@@ -348,11 +372,12 @@ def copy_back_job_results_from_run(self, slotno: int) -> None:
                 else:
                     rfsname = """{}/sim_slot_{}/{}""".format(dest_sim_dir, simserverindex, rfsname)
 
-                run("""sudo mount {blockfile} {mntpt}""".format(blockfile=rfsname, mntpt=mountpoint))
-                with warn_only():
-                    # ignore if this errors. not all rootfses have /etc/sysconfig/nfs
-                    run("""sudo chattr -i {}/etc/sysconfig/nfs""".format(mountpoint))
-                run("""sudo chown -R centos {}""".format(mountpoint))
+                mount(rfsname, mountpoint, f"{dest_sim_dir}/sim_slot_{simserverindex}")
+                if has_sudo():
+                    with warn_only():
+                        # ignore if this errors. not all rootfses have /etc/sysconfig/nfs
+                        run("""sudo chattr -i {}/etc/sysconfig/nfs""".format(mountpoint))
+                    run("""sudo chown -R $(whoami) {}""".format(mountpoint))
```

Comment:
This shouldn't matter for `guestmount` right? Even though it is mounted with `root` permissions you can still R/W within the area.

Reply:
no clue. if it's working without it in your tests then just leave as is for now.